Library/Formula/vault.rb

```diff
@@ -0,0 +1,35 @@
+require "language/go"
+
+class Vault < Formula
+  homepage "https://vaultproject.io/"
+  version "0.1.0"
+  url "https://github.com/hashicorp/vault.git"
+
+  head "https://github.com/hashicorp/vault.git"
+
+  depends_on "go" => :build
+
+  def install
+    gopath = buildpath/"../gopath"
```

Comment:
Unfortunately, the build scripts for this tool require that the git repo be placed within a subdirectory of the `gopath`.  Other `go` scripts use a symlink to achieve this, but this tool's build scripts are more persnickety and I have to place the `gopath` outside of the `buildpath` and then copy it into place.  For the copy to work, the `gopath` can't be within the `buildpath` because it'd be a recursive copy.

My solution here works, but I'm happy to change it if there's a better alternative.